.github/workflows/scripts/check-codeowners.sh

```diff
@@ -63,7 +63,7 @@ check_component_existence() {
   do
     if [[ $line =~ ^[^#\*] ]]; then
       COMPONENT_PATH=$(echo "$line" | cut -d" " -f1)
-      if [ ! -d "$COMPONENT_PATH" ]; then
+      if [ ! -e "$COMPONENT_PATH" ]; then
```

Comment:
Why do we need this change? Also sounds like a candidate for a separate PR :smile:

Reply:
it's because the codeowners file validation fails if the file path offered is not a folder right now. Yes, I'll get an other PR for it, good point.

Reply:
https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/29737